### PR TITLE
Rename example project to RiotSharp.Example

### PR DESCRIPTION
**Pull request by Oucema90**
This is my first contribution to this project :)

I have renamed the project to RiotSharp.Example asked in [#441](https://github.com/BenFradet/RiotSharp/issues/441)

### CODE REVIEW

File: RiotSharp.sln [hunk 1]
```diff
@@ -1,12 +1,10 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26430.4
+# Visual Studio 14
+VisualStudioVersion = 14.0.25420.1
```
Comment (BenFradet) [inline RiotSharp.sln]: could you leave this as is?

**Comment by JanOuborny** [comment]
@Oucema90  Why closing it?

**Comment by Oucema90** [comment]
@JanOuborny It's reopned now; I closed it to handle the changes requested by @BenFradet 
@BenFradet The suggested changes are done.

**Comment by JanOuborny** [comment]
@Oucema90 You don't need to close it for applying requested changes.

**Pull request merged**